TaskSharper.Calender.WPF/Events/Resources/WeekChangedEnum.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TaskSharper.Calender.WPF.Events.Resources
+{
+    public enum WeekChangedEnum
```

Comment:
Possibly change name to "ChangeWeekEnum"?

Reply:
Agree. Done